### ISSUE TITLE
datepicker: make userEvents respond well in test

### ISSUE THREAD

**Issue by jaketrent** [pluralsight/design-system#1469]
Two separate issues:
1) userEvent.tab() does not work as expected when tabbing between mm, dd and yyyy. It always focuses back to the body (I’ve also tried using focusTraps to no avail)
2) userEvent.type() for dd and yyyy is not working (even when not tabbing through) (edited) 

See:
https://pluralsight.slack.com/archives/C70HPSJPP/p1605804091239800?thread_ts=1605632188.221700&cid=C70HPSJPP

**Comment by jaketrent** [comment]
Team triage: Will do, need to figure out why it's working in browser but not in test.
